aea/test_tools/test_cases.py

```diff
@@ -547,15 +568,16 @@ def get_wealth(cls, ledger_api_id: str = DEFAULT_LEDGER) -> str:
         return str(cls.last_cli_runner_result.stdout_bytes, "utf-8")
 
     @classmethod
-    def replace_file_content(cls, src: Path, dest: Path) -> None:
+    def replace_file_content(cls, src: Path, dest: Path) -> None:  # pragma: nocover
         """
         Replace the content of the source file to the dest file.
+
         :param src: the source file.
         :param dest: the destination file.
         :return: None
         """
         assert src.is_file() and dest.is_file(), "Source or destination is not a file."
-        src.write_text(dest.read_text())
+        dest.write_text(src.read_text())
```

Comment:
this changes the direction; check where the functionality is used...

Reply:
it used in documentation test, but it passes
also it looks like we should write src file content to destination file content not the opposite.

Reply:
can you check the logic i applied is ok?